src/frontend/src/lib/components/dapps/DappModal.svelte

```diff
@@ -0,0 +1,84 @@
+<script lang="ts">
+	import { Html, Modal } from '@dfinity/gix-components';
+	import { nonNullish } from '@dfinity/utils';
+	import DappTags from '$lib/components/dapps/DappTags.svelte';
+	import IconGitHub from '$lib/components/icons/IconGitHub.svelte';
+	import IconTwitter from '$lib/components/icons/IconTwitter.svelte';
+	import ExternalLink from '$lib/components/ui/ExternalLink.svelte';
+	import ExternalLinkIcon from '$lib/components/ui/ExternalLinkIcon.svelte';
+	import Img from '$lib/components/ui/Img.svelte';
+	import ImgBanner from '$lib/components/ui/ImgBanner.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import type { DappDescription } from '$lib/types/dappDescription';
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+
+	export let dAppDescription: DappDescription;
+	const formattedUrl = new URL(dAppDescription.website).hostname.replace('www.', '');
```

Comment:
1. This can crash an error

2. Why removing `www`? 

3. nitpick: for coding style consistency prefer destructing objects

Reply:
1. I see your point, we validate the data with zod to be a valid URL. So we would expect someone to use this component with other data than provided by the json in order to provide an invalid URL. I can handle this case gracefully by not rendering any links to this app - Which makes this modal a pure info modal, but I think this is fine for this edge case.
2. matching the design
3. done

Reply:
made it reactive

Reply:
1. We don't have tests in place for either the types using Zod or this component. Therefore no certainty that this won't break someday. Please handle any potential errors.

2. Can you please review this with the designer?

- I assume that if a dapp has chosen to use www, they most likely want to communicate using that URL.
- If we decide to ignore their preferred URL, but a non-www URL is available, we can configure this at the metadata level.
- Given the critical importance of the hostname on the IC, particularly with respect to Internet Identity, I believe we should avoid altering the URL of projects.

Reply:
It's only about the visual appearance of the url, the link in the href is not touched by this formatting. I already checked with the designer: "Cool, let's then make it different. I was interested in cleaner look, so no https:// for sure, we can also try and drop www"

Reply:
"Again, given the importance of hostnames on the IC, I personally would not mess with those, but if that's the wish, then let's not solve it at the code level and instead use a URL without 'www' in the metadata.

Reply:
Hm ok, as he stated 'try and drop www'. I go with your feedback. thanks